lib/identity/stateTransitions/identityCreateTransition/applyIdentityCreateTransitionFactory.js

```diff
@@ -43,7 +43,7 @@ function applyIdentityCreateTransitionFactory(
       .map((publicKey) => publicKey.hash());
 
     await stateRepository.storeIdentityPublicKeyHashes(
-      identity.getId(),
+      identity.getId().toString(),
```

Comment:
We agreed to pass params as EncodedBuffer/Buffer to state repository and validation functions.